src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -2352,6 +2352,39 @@ public class MyComponent : ComponentBase
 
         #region Event Handlers
 
+        [Fact]
+        public void Component_WithImplicitLambdaEventHandler()
```

Comment:
I'm guessing we didn't have existing tests of this because it doesn't work well :( - that's something we need to think harder about in the future.

Reply:
> I'm guessing we didn't have existing tests of this because it doesn't work well :( - that's something we need to think harder about in the future.

Ya definitely. We were lackluster in testing more complex TagHelper attribute scenarios in the past, we need to be more complete here moving forward.